Tests/Tests.xcodeproj/project.pbxproj

```diff
@@ -345,37 +312,41 @@
 		};
 		432B533A1AAE423E00843E69 /* Debug */ = {
 			isa = XCBuildConfiguration;
-			baseConfigurationReference = CCBAE26EBA1DD5C4D8EAF9A6 /* Pods-OS X Tests.debug.xcconfig */;
 			buildSettings = {
+				CLANG_ENABLE_MODULES = YES;
 				COMBINE_HIDPI_IMAGES = YES;
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(DEVELOPER_FRAMEWORKS_DIR)",
 					"$(inherited)",
 				);
 				INFOPLIST_FILE = Info.plist;
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks @loader_path/../Frameworks";
 				SUPPORTED_PLATFORMS = macosx;
+				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_VERSION = 4.2;
```

Comment:
Do we have xcconfig files for the tests? If so, shouldn't these settings be set there?